spec/apm-spec.js

```diff
@@ -0,0 +1,10 @@
+"use babel";
+import { spawnSync } from "child_process";
+
+describe("apm", () => {
+	it("installs the dependencies in the correct test path", function () {
+		spawnSync("apm install atom-ide-base", { shell: true, stdio: "inherit", encoding: "utf8" });
```

Comment:
Now it doesn't install

Reply:
Maybe we should skip this test unless `process.env.CI` is set.

Reply:
Actually it should install into the temp directory so it shouldn't matter if it is already installed